lib/app_perf_rpm/introspector.rb

```diff
@@ -0,0 +1,30 @@
+module AppPerfRpm 
+  class Introspector
+
+    VALID_RUNNERS = %w(Puma Unicorn Sidekiq Sinatra Resque Webbrick)
```

Comment:
This is very incomplete and I'm worried there are a ton of edge cases. I figured I'd get something started and we iterate. I'm testing through it now. 
Use cases I'm focused on:
1. When running rake (assets,etc) it doesn't load the agent at all.
2. Let user declaratively turn off agent.